lib/membrane_http_adaptive_stream/sink.ex

```diff
@@ -171,13 +128,13 @@ defmodule Membrane.HTTPAdaptiveStream.Sink do
     state =
       options
       |> Map.from_struct()
-      |> Map.drop([:manifest_name, :manifest_module])
       |> Map.merge(%{
         storage: Storage.new(options.storage),
-        manifest: %Manifest{name: options.manifest_name, module: options.manifest_module},
-        awaiting_first_segment: MapSet.new(),
-        # used to keep track of partial segments that should get merged
-        track_to_partial_segments: %{}
+        manifest: %Manifest{
+          name: options.manifest_config.name,
+          module: options.manifest_config.module
+        },
+        awaiting_first_segment: MapSet.new()
```

Comment:
couldn't this be handled by manifest?

Reply:
this is only some kind of flag that helps send `playlist_playable` only once. I will change its name to be less confusing.